htdocs/core/tpl/ajaxrow.tpl.php

```diff
@@ -80,7 +80,12 @@ function() {
 						console.log("tableDND end of ajax call");
 						if (reloadpage == 1) {
 							//console.log('<?php echo $urltorefreshaftermove.' - '.$_SERVER['PHP_SELF'].' - '.dol_escape_js($_SERVER['QUERY_STRING']); ?>');
-							location.href = '<?php echo dol_escape_js(empty($urltorefreshaftermove) ? ($_SERVER['PHP_SELF'].'?'.dol_escape_js($_SERVER['QUERY_STRING'])) : $urltorefreshaftermove); ?>';
+							<?php
+							$redirectURL = empty($urltorefreshaftermove) ? ($_SERVER['PHP_SELF'].'?'.dol_escape_js($_SERVER['QUERY_STRING'])) : $urltorefreshaftermove;
+							// remove action parameter from URL
+							$redirectURL = preg_replace('/[&?]action=[^&]*/', '', $redirectURL);
```

Comment:
param1=a&action=abc&param2=b will be replaced with
param1=aparam2=b   (& between param 1 and 2 is lost)